pkg/controller/deployable/template.go

```diff
@@ -94,11 +95,22 @@ func (r *ReconcileDeployable) createManagedDeployable(cluster types.NamespacedNa
 	} else {
 		if !reflect.DeepEqual(original, existingdeployable) {
```

Comment:
as discussed, this is the problem coz spec.template is []byte. 
please replace this with right way to compare.